data_model/src/lib.rs

```diff
@@ -368,6 +382,70 @@ pub mod parameter {
         }
     }
 
+    /// Convenience tool for setting parameters
+    #[derive(Default)]
+    pub struct ParametersBuilder {
+        parameters: Vec<Parameter>,
+    }
+
+    /// Error associated with parameters builder
+    #[derive(From, Debug, Display, Copy, Clone)]
+    pub enum ParametersBuilderError {
+        /// Error emerged during parsing of parameter id
+        Parse(ParseError),
+    }
+
+    #[cfg(feature = "std")]
+    impl std::error::Error for ParametersBuilderError {}
+
+    impl ParametersBuilder {
+        /// Construct [`Self`]
+        pub fn new() -> Self {
+            Self::default()
+        }
+
+        /// Add [`Parameter`] to self
+        ///
+        /// # Errors
+        /// - [`ParameterId`] parsing failed
+        pub fn parameter(
```

Comment:
```suggestion
        pub fn add_parameter(
```

`parameter` should be reserved for getter names